Automattic/wp-calypso
# Comment editing: fix rendering comments before they're edited

## Proposed Changes

Before the user edits a comment, they have to see it rendered. This allows Calypso to render blocks nicely and safely. All the blocks styles are locked to a single div and only effect that.

## Testing Instructions
- Head to /comments/all.
- Test viewing a plan text comment.
- Test viewing block comments with as many blocks as possible. Make sure they render nicely. Round images should be round, and all other customizations! Bold, italic, etc..
